kubernetes/start

```diff
@@ -71,5 +76,23 @@ fi
 sudo mkdir -p $PWD/tmp/mariadb-socket
 sudo chmod 777 $PWD/tmp/mariadb-socket
 
+if docker images | grep "^sles12/velum" &> /dev/null ; then
+  log "Velum development image is present"
+else
+  log "Building Velum development image..."
+  CONTAINER_BUILD=`docker run -v $(readlink -f $PWD/..):/srv/velum -d docker-testing-registry.suse.de/sles12/velum:development \
```

Comment:
shouldn't it be the production image here? `docker-testing-registry.suse.de/sles12/velum:0.0`